qa/rpc-tests/forknotify.py

```diff
@@ -79,30 +78,6 @@ def run_test(self):
             v = self.nodes[0].getblock(str(i))['version']
             assert_equal(v, UP_VERSION)
 
-        # -alertnotify should trigger on the 51'st,
```

Comment:
Lines 46/7: I may be wrong here, but I do not see the option -blockversion among those displayed by zend binary. Am I wrong or this option is somehow outdated/unused?

Reply:
It is a regtest only option, I see it can be displayed via:
  src/zend -regtest --help -help-debug=1